appengine-skeleton-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -27,15 +27,16 @@
         <include>**/*.launch</include>
       </includes>
     </fileSet>
-    <fileSet encoding="UTF-8">
+    <fileSet filtered="true" encoding="UTF-8">
       <directory></directory>
       <includes>
-        <include>nbactions.xml</include>
+        <include>pom.xml</include>
```

Comment:
It did, but once we remove nbactions.xml, the lines look weird.  This made things explicit.